src/index.ts

```diff
@@ -46,6 +46,9 @@ export function oas(cfg: Partial<Config>): koa.Middleware {
         body: (ctx.request as RequestWithBody).body,
       })
     } catch (err) {
+      if (config.errorHandler) {
+        config.errorHandler(err);
+      }
```

Comment:
If the `errorHandler` doesn't throw an Error, the code will keep going here with the `ctx.throw()` though.
I think we can have a `defaultErrorHandler` which is like
```
const defaultErrorHandler(err) {
  if (err instanceOf ChowError) {
      ctx.throw...
   } else { throw err}
}
```

Reply:
Make sense to me.